docs/python_gpu.rst

```diff
@@ -30,32 +27,22 @@ We then proceed to setting up a few parameters.
     N = 200000
 
     # generate positions for the nonuniform points and the coefficients
-    x = 2 * np.pi * np.random.uniform(size=M)
-    c = (np.random.standard_normal(size=M)
-         + 1J * np.random.standard_normal(size=M))
-
-Now that the data is prepared, we need to set up a cuFINUFFT plan that can be executed on that data.
-
-.. code-block:: python
-
-    # create plan
-    plan = cufinufft.Plan(1, (N,), dtype=np.float64)
-
-    # set the nonuniform points
-    plan.setpts(to_gpu(x))
+    x_gpu = 2 * cp.pi * cp.random.uniform(size=M)
+    c_gpu = (cp.random.standard_normal(size=M)
+             + 1J * cp.random.standard_normal(size=M))
 
-With everything set up, we are now ready to execute the plan.
+Now that the data is prepared, we can call cuFINUFFT to compute the transform of the ``M`` source point onto a grid of ``N`` modes
```

Comment:
point -> points
transform from the ...  not transform of the...

Reply:
Done.